src/routers/handlers/check_your_answers/check.your.answers.ts

```diff
@@ -45,4 +53,13 @@ export class CheckYourAnswersHandler extends GenericHandler {
 
         return PrefixedUrls.CONFIRMATION;
     }
+
+    private getAccountsTypeFullName(session: Session | undefined) {
+        const typeOfAccounts = must(getPackageType(session));
+        const accountsType = getAccountType(typeOfAccounts);
```

Comment:
I know you didn't write it, but can we correct the name of this function to getAccountsType